### ISSUE TITLE
Documentation - path is wrong for insert-user when installed from .deb

### ISSUE THREAD

**Issue by KyleMaas** [ssbc/go-ssb-room#326]
When go-ssb-room is installed from the .deb file, the sample command for insert-user should be:

```
sudo ./insert-user -repo "/var/lib/go-ssb-room" "@Bp5Z5TQKv6E/Y+QZn/3LiDWMPi63EP8MHsXZ4tiIb2w=.ed25519"
```

**Comment by decentral1se** [comment]
Yeh it'd be great to show the differences in a dedicated section for the .deb install alongside https://github.com/ssbc/go-ssb-room/blob/master/docs/deployment.md#docker--docker-compose :+1:

**Comment by KyleMaas** [comment]
Fixed with #331
